README.md

```diff
@@ -1,4 +1,4 @@
-# MinIO Operator
+# MinIO Operator:
```

Comment:
?

Reply:
shared already a way to send an empty commit :).